### ISSUE TITLE
Error: can't evaluate field enable in type bool

### ISSUE THREAD

**Issue by Nostalgiaaaa** [dillonzq/LoveIt#275]
刚刚下载了v0.2，hugo serve 启动出现了：
```
Building sites … ERROR 2020/04/28 15:48:27 render of "page" failed: "F:\Novelin\myblog\themes\LoveIt\layouts\posts\single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 15:48:27 render of "page" failed: "F:\Novelin\myblog\themes\LoveIt\layouts\posts\single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 15:48:27 render of "page" failed: "F:\Novelin\myblog\themes\LoveIt\layouts\posts\single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 15:48:27 render of "page" failed: "F:\Novelin\myblog\themes\LoveIt\layouts\posts\single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
Built in 323 ms
Error: Error building site: failed to render pages: render of "page" failed: "F:\Novelin\myblog\themes\LoveIt\layouts\posts\single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
```
不知道是什么问题...

**Comment by huanauh** [comment]
同样的问题，应该是个bug。
```
Building sites … ERROR 2020/04/28 16:02:02 render of "page" failed: "/root/hugo/themes/LoveIt/layouts/posts/single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 16:02:02 render of "page" failed: "/root/hugo/themes/LoveIt/layouts/posts/single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 16:02:02 render of "page" failed: "/root/hugo/themes/LoveIt/layouts/posts/single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
ERROR 2020/04/28 16:02:02 render of "page" failed: "/root/hugo/themes/LoveIt/layouts/posts/single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
Built in 2599 ms
Error: Error building site: failed to render pages: render of "page" failed: "/root/hugo/themes/LoveIt/layouts/posts/single.html:7:21": execute of template failed: template: posts/single.html:7:21: executing "content" at <$params.toc.enable>: can't evaluate field enable in type bool
```

**Comment by anup92k** [comment]
Hello,

Same problem here but the exemple site is running well...

```bash
cd themes/LoveIt
hugo server --themesDir ../..
```

```shell
Building sites … WARN 2020/04/28 10:03:36 

Current environment is "development". The "comment system", "CDN" and "fingerprint" will be disabled.
当前运行环境是 "development". "评论系统", "CDN" 和 "fingerprint" 不会启用.


                   | EN  | ZH-CN | FR   
-------------------+-----+-------+------
  Pages            |  39 |    39 |  39
  Paginator pages  |   0 |     0 |   0  
  Non-page files   |   0 |     0 |   0
  Static files     | 115 |   115 | 115
  Processed images |   0 |     0 |   0
  Aliases          |  11 |    10 |  10
  Sitemaps         |   2 |     1 |   1
  Cleaned          |   0 |     0 |   0

Built in 1482 ms
Watching for changes in [...]\themes\LoveIt\{archetypes,assets,exampleSite,i18n,layouts,static}
Watching for config changes in [...]\themes\LoveIt\exampleSite\config.toml
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

**Comment by dillonzq** [comment]
Hi, Everyone!
Version 0.2.0 is an incompatible major version update.
Sorry, some configuration and front matter parameters have changed due to some reasons.
Please check the latest [documentation](https://hugoloveit.com/theme-documentation-basics/#site-configuration) to change the parameters.

**Comment by dillonzq** [comment]
@huanauh @Nostalgiaaaa `toc` has been a global parameter in the `page` part of the site configuration file.
The front matter has the same structure with the site configuration.

**Comment by anup92k** [comment]
> 
> 
> @huanauh @Nostalgiaaaa `toc` has been a global parameter in the `page` part of the site configuration file.
> The front matter has the same structure with the site configuration.
> 
> ![image](https://user-images.githubusercontent.com/30786232/80464505-7e7fcf00-896c-11ea-976f-8d1b70df9f87.png)
> 
> You only need to set different value in the front matter or don't set it.

That was it !
Just removed all the "toc: true" to my posts files (` sed -i '/toc: true/d' *.md`) and it works as expected.

Thanks !

**Comment by Nostalgiaaaa** [comment]
@huanauh  成功了！把所有文章的 toc 和 math 移除，可以启动了。Thanks @anup92k @dillonzq

**Comment by Nostalgiaaaa** [comment]
还有如果用 `hugo serve -e production` 启动，还需要将 front matter 的 content 移除

**Comment by Nostalgiaaaa** [comment]
原来是这样，看了 [content](https://hugoloveit.com/theme-documentation-content/)，发现 front matter 里的 toc 写法变了，以及 math、comment 等等。
```yaml
toc:
  enable: true
  auto: true
math:
  enable: true
  # ...
comment:
  enable: true
  # ...
```

**Comment by dillonzq** [comment]
@Nostalgiaaaa 与 `config.toml` 的参数保持一致，既能更好地进行全局配置，又能单独设置每个页面的具体参数，而不仅仅是一个布尔值。

**Comment by huanauh** [comment]
Thanks. @dillonzq @anup92k

**Comment by dillonzq** [comment]
Fixed in [v0.2.7](https://github.com/dillonzq/LoveIt/releases/tag/v0.2.7)
